plugins/com.google.cloud.tools.eclipse.appengine.login/plugin.xml

```diff
@@ -1,5 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+   <extension
+         point="org.eclipse.ui.commands">
+      <category
+            id="com.google.cloud.tools.eclipse.appengine.login.commands.category"
+            name="Google Login Category">
```

Comment:
Do we need "appengine" in the id? Login is for the GCP not only App Engine right?